### PR TITLE
Add audioswitcher manifest

### PR DESCRIPTION
**Pull request by goostleek**
https://audioswit.ch/er

### CODE REVIEW

File: bucket/audioswitcher.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+            "AudioSwitcher.exe",
+            "Audio Switcher"
+        ]
+    ],
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/xenolightning/AudioSwitcher_v1/releases/download/$version/AudioSwitcher.zip"
+    },
+    "pre_install": "Stop-Process -Name AudioSwitcher -Verbose -ErrorAction Ignore",
+    "post_install": "Start-Process AudioSwitcher.exe -Verbose -NoNewWindow"
```
Comment (Ash258) [inline bucket/audioswitcher.json]: Do not start it automatically. No all users would want to start applications automatically after installing.

```suggestion
```

File: bucket/audioswitcher.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "url": "https://github.com/xenolightning/AudioSwitcher_v1/releases/download/1.8.0.142/AudioSwitcher.zip",
+    "hash": "e675b85a7c8465e02b58148f8d98b4adab78c3a7423160ca78d1237d8469ace4",
+    "bin": "AudioSwitcher.exe",
+    "shortcuts": [
+        [
+            "AudioSwitcher.exe",
+            "Audio Switcher"
+        ]
+    ],
+    "checkver": "github",
```
Comment (Ash258) [inline bucket/audioswitcher.json]:
```suggestion
    "checkver": {
        "github": "https://github.com/xenolightning/AudioSwitcher_v1/"
    },
```

Comment (goostleek) [reply bucket/audioswitcher.json]: Fixes applied @Ash258.

**Comment by goostleek** [comment]
Fixes applied @Ash258.

**Pull request merged**